ArduCopter/GCS_Mavlink.cpp

```diff
@@ -454,6 +463,7 @@ static const ap_message STREAM_EXTRA3_msgs[] = {
     MSG_VIBRATION,
     MSG_RPM,
     MSG_ESC_TELEMETRY,
+    MSG_LANDING_TARGET,
```

Comment:
I wouldn't add it to a stream. We now support requesting a single message !

Reply:
OK